campaignion_email_to_target/campaignion_email_to_target.module

```diff
@@ -172,6 +174,9 @@ function campaignion_email_to_target_tokens($type, array $tokens, array $data =
   if ($type == 'email-to-target' && !empty($data['email-to-target'])) {
```

Comment:
This should be replaced with `$data['target']` too. We entirely control how data is passed to this function so there is no need to keep the old data around.